tests/fast-integration/compass-test/compass-test.js

```diff
@@ -44,15 +42,15 @@ describe("Kyma with Compass test", async function() {
       return;
     }
 
-    await installer.installKyma({withCompass: true, skipComponents, withCentralApplicationGateway});
+    await installer.installKyma({skipComponents: componentsToSkip, withCompass: true, withCentralAppConnectivity: withCentralAppConnectivity});
```

Comment:
You can leave [shorthand property names notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#new_notations_in_ecmascript_2015)

Reply:
Now it's like this:
```js
await installer.installKyma({withCompass: true, withCentralAppConnectivity: withCentralAppConnectivity, componentsToSkip});
```